code/game/verbs/ooc.dm

```diff
@@ -74,8 +74,8 @@ var/global/admin_ooc_colour = "#b82e00"
 
 			if(holder)
 				if(holder.fakekey)
-					if(C.holder)
-						display_name = "[holder.fakekey]/([src.key])"
+					if(C.holder.rights & R_ADMIN)

```

Comment:
Both of these should actually be `C.holder && C.holder.rights & R_ADMIN`

Reply:
Fixed on OOC, unless I'm mistaken it isn't necessary in msay due to drawing from admins and the if(check_rights(R_ADMIN|R_MOD|R_MENTOR, 0, C.mob)) guaranteeing holder exists. Please inform me if I'm incorrect. And sorry about that, I had a feeling I forgot to check something...